Source/CISQLite3/Public/SQLiteDatabase.h

```diff
@@ -111,6 +111,7 @@ struct SQLiteQueryResult
 	bool Success;
 	FString ErrorMessage;
 	TArray<SQLiteResultValue> Results;
+	int InsertedId = 0;

```

Comment:
This is a functional change - not sure if this is the way you'd like it.. But I really needed the "last insert id" for an inserted record from a regular query (I'm only using this as a C++ module, not using the blueprints), so you might want to check that out.

Reply:
There is a SQLite (litterally, specific to sqlite) query for that `SELECT last_insert_id();` or something. It returns the same as `sqlite3_last_insert_rowid(db)`. (Might be slower, though).
If you could check out how that functions is implemented... if it's trivial, we can (and probably should) keep this, otherwise I recommend the query.